### ISSUE TITLE
subscribeToMore has no previous query data

### ISSUE THREAD

**Issue by bsh314** [awslabs/aws-mobile-appsync-sdk-js#188]
I seems like "subscribeToMore" function is not returning original query response.
Is this a bug or the lack of information about it's functionality?

Here I provide some code:

```javascript
import gql from 'graphql-tag';

export default gql`
query getMsgs($conversationId: ID!, $limit: Int, $nextToken: String) {
  getMsgs(conversationId: $conversationId, limit: $limit, nextToken: $nextToken) {
    __typename
    messages {
      __typename
      id
      ...
    }
  }
}`;

export default gql`
subscription newMsgs($conversationId: ID!) {
  newMsgs(conversationId: $conversationId) {
    __typename
    id
    ...
  }
}`;

    return new Observable(observer => {
       const options = {
          query: getMsgs,
          variables: { conversationId: conversationId }
        };
        const obs = client.watchQuery(options);
        obs.subscribe(({ data }) => {
          messages = data ? data.getMsgs.messages : [];
          observer.next(messages);
          return messages;
        });

        obs.subscribeToMore({
          document: newMsgs,
          variables: { conversationId: conversationId },
          updateQuery: (prev, { subscriptionData: { data: { newMsgs: message } } }) => {
            // This "prev" data is always an empty object "{}"
            console.log('subscribeToMore - prev data:', prev);
          }
        });
      });
```

Thanks for your help!

**Comment by manueliglesias** [comment]
Hi @bsh314 

Are you still experiencing this issue? I tried to replicate with no luck. I can see the `prev` result.

Would it be possible to get a codesandbox or similar showing the issue?

**Comment by elorzafe** [comment]
I will close this for now, but feel free @bsh314 to reopen this issue or to create a new one.

**Comment by ChrisFieldsII** [comment]
I just experienced the same thing with a few different subscriptions where prev was an empty object. I was able to solve it by making sure that my queries, mutations, subscriptions files containing the gql templates all had the exact same return data. I was missing one field (updatedAt) in the mutations and subscriptions file return data where it was specified in the query and this caused prev to be an empty object.
